README.md

```diff
@@ -84,7 +84,7 @@ A `for` loop offers a lot of flexibility, I can modify:
 
 This also implies that I may introduce **errors** and that I have no guarantees about the returned value.
 
-**Quiz**. Is the `for loop` correct?
+**Quiz**. Is the `for loop` correct? (see the [answer here](src/quiz-answers/for-loop.md))
```

Comment:
Maybe add the answers on a separate line, so the diff is more clear (since you aren't changing the question itself, you're adding an answer)

Reply:
Thanks for the feedback. I modified it in a slightly different way than you proposed because having it just below felt like it was not really linked to the quiz. I could not have it displayed on the line above without some spacing.
WDYT?